client/src/components/Chat/ChatHeader.js

```diff
@@ -77,6 +77,7 @@ const ChatHeader = props => {
   const handleChange = (event) => {
     const {name, checked} = event.target;
     setState({ ...state, [name]: checked });
```

Comment:
seems like this logic is going to result in duplicate storage of the toggle checked state (in ChatHeader state and also in the Chat component).

Reply:
I noticed that Switch component itself relies on these states. Removing them from here or trying to pass them as props from parent Chat component is not allowing it to be flipped for some reason. Let me know if you have suggestions